### DISCUSSION TITLE
On-Demand Revalidation

### DISCUSSION THREAD

**Discussion by GiantappMan** [withastro/roadmap#320] [Proposal]
Is it possible to revalidate special pages as needed?
For example I changed something in cms and then called `API/Update/about`,
Astro rebuilds only the About page instead of rebuilding all pages.

<https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration#using-on-demand-revalidation>

**Comment by panwauu** [comment]
ISR is not supported but with Astro currently. Although you can achieve something similar with cache headers:
See also https://discord.com/channels/830184174198718474/830184175176122389/1022490330857230367

**Reply by virketrang** [reply]
Could you elaborate a bit on how you would go about implementing on-demand revalidation with Astro?
